package.json

```diff
@@ -49,47 +50,68 @@
     "prettier"
   ],
   "dependencies": {
+    "@typegoose/typegoose": "^9.7.1",
     "bcryptjs": "^2.4.3",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "cross-env": "^7.0.0",
-    "dotenv": "^10.0.0",
-    "express": "^4.17.1",
-    "express-mongo-sanitize": "^2.0.0",
-    "express-rate-limit": "^5.0.0",
-    "helmet": "^4.1.0",
-    "http-status": "^1.4.0",
-    "joi": "^17.3.0",
+    "cross-env": "^7.0.3",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.3",
+    "express-mongo-sanitize": "^2.2.0",
+    "express-rate-limit": "^6.3.0",
+    "helmet": "^5.0.2",
+    "http-status": "^1.5.0",
+    "joi": "^17.6.0",
```

Comment:
I think you can remove this `joi` dependency if you replaced it with `zod` instead.